package.json

```diff
@@ -20,6 +20,7 @@
   "dependencies": {
     "@atomist/automation-client": "0.17.2-20180605032022",
     "@atomist/slack-messages": "^0.12.1",
+    "@jessitron/git-branch-to-npm-tag": "^1.0.1",
```

Comment:
Shouldn't this be deployed under `@atomist`, or just rolled into SDM? Is it more than just a regular expression replace?